es6-classes/example-class.js

```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  contructor() {
```

Comment:
this has a typo in it but you can just omit the `constructor()` as JavaScript will give your class a default one in this case